CMakeLists.txt

```
@@ -326,4 +326,8 @@ endif()
 
 add_subdirectory(docs)
 
-include(CMakeCPack.cmake)
+if(DEFINED ENV{LIB_FUZZING_ENGINE})
+  add_subdirectory(fuzz)
+endif()
+
+include(CmakeCPack.cmake)
```

There is an upper-to-lower case change here that causes the Linux builds to break. The macOS and Windows builds do not suffer from this because those are on a case-insensitive file system.

```suggestion
include(CMakeCPack.cmake)
```

with this change, it is fine: https://dev.azure.com/erikboasson/Cyclone%20DDS/_build/results?buildId=258&view=results (two builds are taking ridiculously long, usually that ends up causing some timeouts in the tests).